include/OpenColorIO/OpenColorTransforms.h

```diff
@@ -1115,9 +1082,21 @@ class OCIOEXPORT LookTransform : public Transform
     // look specification. (And forward is assumed in the absence of either)
     void setLooks(const char * looks);
 
+    bool getSkipColorSpaceConversion() const;
```

Comment:
Worth a comment to explain what this does.

Reply:
adding comment

Reply:
Thanks!